index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+    <style>
+      .container{
+  display: flex;
+}
+    </style>
```

Comment:
This CSS styling should be inside the external file.

Reply:
- [ ] While this is a nice comment, however, it can be improved by explaining the `why` and kindly addressing it as a suggestion with a polite tone instead of a command, also it will be great if you provide resources about best practices on this concept so the student will learn something new 🧑‍🎓